pytorch_lightning/metrics/functional/ssim.py

```diff
@@ -19,17 +19,17 @@
 from torch.nn import functional as F
 
 
-def _gaussian_kernel(channel, kernel_size, sigma, device):
-    def _gaussian(kernel_size, sigma, device):
-        gauss = torch.arange(
-            start=(1 - kernel_size) / 2, end=(1 + kernel_size) / 2, step=1, dtype=torch.float32, device=device
-        )
-        gauss = torch.exp(-gauss.pow(2) / (2 * pow(sigma, 2)))
-        return (gauss / gauss.sum()).unsqueeze(dim=0)  # (1, kernel_size)
+def _gaussian(kernel_size: int, sigma: int, dtype: torch.dtype, device: torch.device):
+    dist = torch.arange(start=(1 - kernel_size) / 2, end=(1 + kernel_size) / 2, step=1, dtype=dtype, device=device)
+    gauss = torch.exp(-pow(dist / sigma, 2) / 2)
```

Comment:
```suggestion
    gauss = torch.exp(-torch.pow(dist / sigma, 2) / 2)
```

Reply:
Also agree with this, otherwise LGTM, thanks!